QUESTIONS.md

```diff
@@ -3,3 +3,10 @@
 What questions can you come up with?
 
 ## Cohort Questions
+1 Need more help on exercies on writing code to connect to the front end.
+2 Some challenges to create a code to connect to the databse and insert a row or create a column in a database.
+3 Need advise on how to maintain cotinuity of thought while working on an particular project and keeping in mind the concepts and how they are all connected.
+4 How do we all stay connected after this training batch is over.
+5 Who would be available after this 18 weeks is over in case we have some technical questions.
```

Comment:
Me for sure if you can't ever reach a TA.

Reply:
Nice of you Hollis